src/main/common/router/routablePath.ts

```diff
@@ -1,5 +1,11 @@
+import { StringUtils } from 'utils/stringUtils'
+
 const pathParameterRegex = /\/:[^\/]+/g
 
+function isValidParameterValue (parameterValue: string): boolean {
+  return !(StringUtils.isBlank(parameterValue) || parameterValue === 'undefined' || parameterValue === 'null')
```

Comment:
What's the purpose of `StringUtils.isBlank` if you are still checking for null and undefined later? Might as well check string length.

Reply:
`StringUtils.isBlank` checks for `undefined` and `null` and then later we check for `'undefined'` and `'null'` (stings).